cmd/testcmd.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/speakeasy-api/speakeasy/internal/model"
+	"github.com/speakeasy-api/speakeasy/internal/model/flag"
+	"github.com/speakeasy-api/speakeasy/internal/testcmd"
+)
+
+// testCmd is the command for running target tests.
+var testCmd = &model.ExecutableCommand[testCmdFlags]{
+	Usage:            "test",
+	Short:            "For each workflow target, starts the mock API server and runs testing.",
+	Run:              testCmdRun,
+	RunInteractive:   testCmdRunInteractive,
+	RequiresAuth:     true,
+	UsesWorkflowFile: true,
+	Flags: []flag.Flag{
+		flag.BooleanFlag{
+			Name:        "disable-mockserver",
```

Comment:
What about just `--mock` / `--no-mock`

Reply:
I'm amenable to the naming if there's consensus/reasoning. It's called `mockserver` in the generator and in the `gen.yaml`. We want it to be enabled by default so testing working out of the box. 👍

Reply:
I dropped a note in Slack about this for potential followup, but quite low risk to merge as-is since its not actually wired up for a few days. Love a good naming conversation 😄